### PR TITLE
Use config files to configure personal info and cv events

### PR DESCRIPTION
**Pull request by jankapunkt**
In the long run it makes more sense to define these info in a config file. Every template (except infographics) includes them using `\input` and retrieves them via `\meta<name>` and `\eventsLoad` or `\eventsRender`.

The render of a `\cvevent` is still part of each template. The whole change should be backward compatible.

### CODE REVIEW

**Comment by jankapunkt** [comment]
@watchers Anyone wants to review or comment on this?

**Comment by CsatiZoltan** [comment]
I will soon dig into the details of the source codes and give my opinion.

**Comment by CsatiZoltan** [comment]
Using global config files makes the content separated from the style, except when special templates (such as *infographics*, or the upcoming *academic*) are used. Apart from this, do you think the end-user can edit the CSV file more easily than the original tex file? I think - if sticking to the original concept of being self-contained and easy-to-understand - the current version in the master branch is easier to comprehend. What if the user breaks the CSV file? Then I guess he/she receives a rather cryptic error message from the `datatool` Latex package. In my opinion, the idea of using config files is fine, but not for substantially different template files. Or you could leave it for the "industrial" CVs, but then filling in the personal data will be different for the *classic*, *modern*, etc. templates and for the *infographics* template.

**Comment by CsatiZoltan** [comment]
Yes, I would definitely keep it. You could set a pointer to this branch in the README of the master branch.

File: README.md [hunk 1]
```diff
@@ -114,6 +114,18 @@ This repo also contains a `texlive.profile` file in the project root, that can b
  * Run `pdflatex` (build/compile) 
  - The `main.pdf` should show the output.
 
+## New: Configuration files
+
+To ease bilding multiple templates at once, the project introduces the `config` folder, where personal and event data can be configured.
```
Comment (virgilwashere) [inline README.md]:
```suggestion
To ease building multiple templates at once, the project introduces the `config` folder, where personal and event data can be configured.
```

Just a typo.

File: config/config_meta.tex [hunk 2]
```diff
@@ -0,0 +1,33 @@
+% ======================================================================================================================
+%   This is the definition of your personal data, which we define here with pattern \meta<var>
+%   We use \def in order to make these variables overridable wihtin your templates, see
+%   https://tex.stackexchange.com/questions/655/what-is-the-difference-between-def-and-newcommand
+%
+%   Include this file in your tempalte usinf \input in order to include the definitions.
```
Comment (virgilwashere) [inline config/config_meta.tex]:
```suggestion
+% Add this file to your template using \input to use the definitions.
```

Typos.

**Pull request closed without merging**